### ISSUE TITLE
NOTE: Redirects are currently not supported in Windows or MacOs.

### ISSUE THREAD

**Issue by awei0116** [MzeroMiko/VMamba#103]
When I use the  commands "python -m torch.distributed.launch --nnodes=1 --node_rank=0 --nproc_per_node=8 --master_addr="127.0.0.1" --master_port=29501 main.py --cfg </path/to/config> --batch-size 128 --data-path </path/of/dataset> --output /tmp" to train VMamba models for classification，it occured that. Where should I revise to run it in Windows?Looking forward to your responding !
![9e18db70117575f7120b9871f2f9e27](https://github.com/MzeroMiko/VMamba/assets/164145551/2a342abd-47cb-4f6a-a819-3b9f8e554f6f)

**Comment by MzeroMiko** [comment]
I don't think you are using `python -m torch.distributed.launch --nnodes=1 --node_rank=0 --nproc_per_node=8 --master_addr="127.0.0.1" --master_port=29501 main.py`, instead, it seems like you are using `python` directly. So you can try again to see what happens next.

By the way, you can refer to #95 for running VMamba in windows.

**Comment by awei0116** [comment]
I tried the command"python -m torch.distributed.launch --nnodes=1 --node_rank=0 --nproc_per_node=8 --master_addr="127.0.0.1" --master_port=29501 main.py" again and met that question again
![Uploading 9b820444305ba0a29ad245c98f39ec3.png…]()
... When I install selective_scan on windows,I also met same question , I have revised according to #95 and successfully solved it.

**Comment by awei0116** [comment]
![Uploading 9b820444305ba0a29ad245c98f39ec3.png…]()

**Comment by YangQu123** [comment]
Hello, have you solved the problem? I had the same problem.

**Comment by awei0116** [comment]
No.it does not work...
&nbsp;
&nbsp;
------------------&nbsp;Original&nbsp;------------------
From: ***@***.***&gt;;
Date: &nbsp;Thu, Jun 13, 2024 02:15 PM
To: ***@***.***&gt;; 
Cc: ***@***.***&gt;; ***@***.***&gt;; 
Subject: &nbsp;Re: [MzeroMiko/VMamba] NOTE: Redirects are currently not supported in Windows or MacOs. (Issue #103)

&nbsp;



 
Hello, have you solved the problem? I had the same problem.
